packages/commerce-sdk-react/src/hooks/ShopperBaskets/mutation.ts

```diff
@@ -274,18 +293,15 @@ export const getCacheUpdateMatrix = (customerId: string | null) => {
             const basketId = params.parameters.basketId
 
             return {
-                ...updateBasketQuery(basketId),
-                ...invalidateCustomerBasketsQuery(customerId)
+                ...updateBasketQuery(basketId, response)
             }
         },
         createBasket: (
             params: Argument<Client['createBasket']>,
             response: DataType<Client['createBasket']>
         ): CacheUpdateMatrixElement => {
-            const basketId = response.basketId
-
             return {
-                ...updateBasketQuery(basketId),
+                // we want to re-fetch basket data in this case to get the basket total and other baskets data
```

Comment:
Doesn't the Create Basket endpoint return a basket? Could we not just use that data as we do with the other endpoints to `updateBasketQuery(basketId, response)` ?

Reply:
Ok... I can get on board with that. I thing there will be room for improvement/optimization here tho. But we can address that in the next feature update as originally planned. 👍